nservicebus/operations/performance-counters-slaviolation.include.md

```diff
@@ -0,0 +1,16 @@
+## SLA violation countdown
+
+**Counter Name:** `SLA violation countdown`
+
+**Added in:** Version 3
+
+Acts as a early warning system to inform on the number of seconds left until the SLA for the endpoint is breached. This gives a system-wide counter that can be monitored without putting the SLA into the monitoring software. Just set that alarm to trigger when the counter goes below X, which is the time that the operations team needs to be able to take actions to prevent the SLA from being breached.
```

Comment:
> This gives a system-wide counter that can be monitored without putting the SLA into the monitoring software.

That sentence is not very clear to me, what do we mean by putting SLA into the monitoring software? Could we simply say that this threshold is applied system-wise, for all endpoints?